### DISCUSSION TITLE
Can`t flim with 10bit GPLOG

### DISCUSSION THREAD

**Discussion by Jupiwai** [gopro/labs#1207] [Q&A]
GoPro13 Labs software. Cant shoot with 10bit, all videos are in 8bit. Use QRcontrol to set features
![image](https://github.com/user-attachments/assets/afe85d16-e7d5-43f2-b8fa-b32de1aa1246)

**Comment by dnewman-gpsw** [comment]
You have set 8-bit in that QR-code.  Are you trying to use the GP-Log feature or Labs LOGB controls?  You shouldn't mix the two.

**Reply by Jupiwai** [reply]
Ooups, wrong screenshot. I used different types of settings and didnt get correct result. I always get 8bit video file. What settings i need to get 10bit video with the log color profile(gp log or logb, not so important)?

**Reply by dnewman-gpsw** [reply]
There is a bug 1.30 (not Labs related), but there is a work around.  Reset your presets, or scan !PRESET.  Set you default "Video" preset to 10-bit, GP Log or whatever. Then when creating new presets, don't press "Create New Presets", instead edit the Video present and scroll to the bottom and select "Save As".  The result is a new preset, that will support 10-bit. There was a bug (it is fixed for the next release) in the "Create New Presets" path that set support to 8-bit only.

If you use $PRES="0,LabsPreset", from a working 10-bit preset, it also works.
